extensions/wikia/CuratedContent/CuratedContentValidatorController.class.php

```diff
@@ -15,6 +16,11 @@ public function __construct() {
 	}
 
 	public function validateSection() {
+		global $wgRequest;
```

Comment:
Well, the difference is that we don't use a global. I don't know if it's better ;) PHP magician needed.